src/Private/Timers.ps1

```diff
@@ -72,7 +72,7 @@ function Invoke-PodeInternalTimer {
     )
 
     try {
-        $global:TimerEvent = @{
+        $script:TimerEvent = @{
```

Comment:
This needs to be `$global:` otherwise the `$TimerEvent` object is `$null`

Reply:
I'm going to check it

Reply:
Roughly the same explanation as I gave for Azure/AWS serverless, it's a scoping issue that seems to cause it, and unless these have `$global:` then scoping makes them `$null`